libcxx/include/__config_site.in

```diff
@@ -42,6 +42,11 @@
 // Hardening.
 #cmakedefine _LIBCPP_HARDENING_MODE_DEFAULT @_LIBCPP_HARDENING_MODE_DEFAULT@
 
+// C libraries
+#cmakedefine LIBCXX_LIBC_LLVMLIBC
+#cmakedefine LIBCXX_LIBC_PICOLIBC
+#cmakedefine LIBCXX_LIBC_NEWLIB
```

Comment:
I'd use `#cmakedefine01` here and then I'd check `#if _LIBCPP_LIBC_NEWLIB` instead of `#if defined(_LIBCPP_LIBC_NEWLIB)`. This can prevent mistakes when coupled with `-Wundef`.

Reply:
Very good suggestion, done - the checks are much cleaner now.